debian/usr.bin.snap-confine

```diff
@@ -1,7 +1,7 @@
 # Author: Jamie Strandboge <jamie@canonical.com>
 #include <tunables/global>
 
-/usr/bin/snap-confine (attach_disconnected) {
+/usr/lib/snapd/snap-confine (attach_disconnected) {
```

Comment:
I'd like to see a spread test for this. The existing spread tests will show when the profile needs updating when snap-confine is running under confinement. However, we need to make sure it is running under confinement. I suggest:
1. install hello-world
2. cp /etc/apparmor.d/usr.bin.snap-confine /tmp
3. add 'audit deny file,' to /tmp/usr.bin.snap-confine
4. apparmor_parser -r /tmp/usr.bin.snap-confine
5. if hello-world.echo ; then exit 1 ; fi

restore consists of:
1. apparmor_parser -r /etc/apparmor.d/usr.bin.snap-confine
2. rm -f /tmp/usr.bin.snap-confine

In this manner, we copy the profile to a temporary location, add a rule that is guaranteed to make the launcher fail to launch, then try to run hello-world. If it succeeds we know the binary attachment is wrong.